### ISSUE TITLE
Shift key not working for key bindings

### ISSUE THREAD

**Issue by manjaroblack** [qutebrowser/qutebrowser#7994]
**Version info**:

qutebrowser v3.0.2
Backend: QtWebEngine 6.5.3, based on Chromium 108.0.5359.220 (from api)
Qt: 6.5.3

https://bpa.st/75LQ

**Does the bug happen if you start with `--temp-basedir`?**:
yes

**Description**
I am unable to use any key bindings that require using the shift key, for example O to open in a new tab. The shift key works everywhere else in qutebrowser except for key bindings. I am not sure if it is relevant or not. I am using a Windows 11 Pro VM. I am RDP'd into the Windows VM from my iPad.

**How to reproduce**

Install qutebrowser in a Windows 11 Pro VM, then RDP into the VM on an iPad.

**Comment by toofar** [comment]
Can you check what `:debug-keytester` says when the widget has focus and you press shift?

**Comment by manjaroblack** [comment]
[image]

When I press shift when it is first open, it just says waiting for keypress. If I type a capital O, the text shows a capital O. It always says NoModifier.

On Nov 9, 2023, at 18:28, toofar ***@***.***> wrote:

﻿

Can you check what :debug-keytester says when the widget has focus and you press shift?

**Comment by toofar** [comment]
Hmm, not sure where to go from there. If we aren't getting the keypresses at all there doesn't seem to be much we can do. But if the shift key is being passed to the web pages that's a little strange - they key presses are forwarded to the pages from the Qt widgets, which is what that key tester is.

**Comment by The-Compiler** [comment]
Yeah, I'm afraid this might be a "not much we can do" situation given the rather exotic setup. However, could you reproduce this with `--temp-basedir -D log-sensitive-keys` and then do `:report` so I can take a look at the full debug logs?
